UMLEditor/UMLEditor/Views/MainWindow.axaml.cs

```diff
@@ -46,14 +72,74 @@ private void HelpB_OnClick(object sender, RoutedEventArgs e)
                 "\nList Classes: List all existing classes" +
                 "\nList Attributes: List all attributes of a class" +
                 "\nList Relationships: List all relationships of a class";
+            
         }
 
         private void List_Classes_OnClick(object sender, RoutedEventArgs e)
         {
-            TextBox outputBox = this.FindControl<TextBox>("OutputBox");
+            
+            // TextBox outputBox = this.FindControl<TextBox>("OutputBox");
+
+            // outputBox.Text = "laksjdflaskjdf";
+
+        }
+
+        private void AddRelationship_OnClick(object sender, RoutedEventArgs e)
+        {
+
+            //User input is taken in from the textbox, validation is done to make sure that what the user entered is valid, add relationship if valid.
+            string input = InputBox.Text;
+            
+            //Split the input into words to use later on.
+            string[] words = input.Split(" ".ToCharArray() , StringSplitOptions.RemoveEmptyEntries);
 
-            outputBox.Text = "laksjdflaskjdf";
+            const string NONEXISTENT_NAME_FORMAT = "Nonexistent class name entered ({0}).";
+
+            if (words.Length == 0)
```

Comment:
I thought we were validating names when we construct the object.  What is the benefit to validating the input through the OnClick event handler?